src/cmd/cmdgazebo.rb.in

```diff
@@ -435,6 +446,11 @@ class Cmd
       # and gui.
       if options['server'] == 0 && options['gui'] == 0
 
+        if plugin.end_with? ".dylib"
+          puts "`ign gazebo` currently only works with the -s argument on macOS."
```

Comment:
How about adding a comment with a link to https://github.com/ignitionrobotics/ign-gazebo/issues/44?

Reply:
added in 0dfa9aa175adf053c068ac087e8311b09e9897ad